superset-frontend/cypress-base/cypress/integration/explore/visualizations/line.test.ts

```diff
@@ -33,6 +33,14 @@ describe('Visualization > Line', () => {
     cy.get('.alert-warning').contains(`"Metrics" cannot be empty`);
   });
 
+  it('should preload mathjs', () => {
+    cy.get('script[src*="mathjs"]').should('have.length', 1);
```

Comment:
The tests you have to show whether or not mathjs has been pre-loaded are great. Do you think it's worth adding a test that switches the chart type in explore, and checks to make sure it _gets_ loaded on-demand? Just doesn't seem there's a test where the length goes from 0 to 1.